modules/update-upgrading-cli.adoc

```diff
@@ -78,6 +56,18 @@ VERSION IMAGE
 4.10.0  quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b
 ----
 
+. Based on your organization requirements, set the channel to `stable-4.10`, `fast-4.10`, or `eus-4.10`. For more information about the upgrade channels, see xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]:
```

Comment:
Can't xref from modules still, unfortunately.

```suggestion
. Based on your organization requirements, set the channel to `stable-4.10`, `fast-4.10`, or `eus-4.10` by running the following command:
```

Reply:
To get around the xref limitation:
1. Open the `updating/updating-cluster-cli.adoc` assembly.
2. Add a line under `include::modules/update-upgrading-cli.adoc[leveloffset=+1]`
3. Put the following on the line immediately after the `include`:
```
[role="_additional-resources"]
.Additional resources
* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
```
It's not ideal, but that is the closest we can get to what you want to do here.

Reply:
Fixed.